manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.locale.en-US.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.Uni
+PackageVersion: 8.7.12
+PackageLocale: en-US
+Publisher: AnyLogic North America
+PublisherUrl: https://the.anylogic.company/
+# PublisherSupportUrl: 
+PrivacyUrl: https://anylogic.com/upload/privacy_policy/anylogic-privacy-policy.pdf
+# Author: 
+PackageName: Anylogic University
```

Comment:
Because they put the version in the middle of the display name, AppsAndFeaturesEntries are needed